springfox-spi/src/main/java/springfox/documentation/spi/service/contexts/RequestMappingContext.java

```diff
@@ -187,6 +187,10 @@ public String getGroupName() {
   public <T extends Annotation> Optional<T> findControllerAnnotation(Class<T> annotation) {
     return handler.findControllerAnnotation(annotation);
   }
+  
+  public List<Annotation> getAnnotations() {
```

Comment:
Does the `findControllerAnnotation` `findAnnotations` not work? I feel like a plugin should request only annotations it knows about. This is very open interface in my opinion.

Reply:
I've deleted that already. The main idea is to make common interface to handle all cases with projections.